src/applications/disability-benefits/all-claims/tests/containers/Missing526Identifiers.unit.spec.jsx

```diff
@@ -98,4 +98,112 @@ describe('Form 526 Missing Identifiers Error Message', () => {
       );
     });
   });
+
+  describe('birlsId Missing', () => {
+    const props = {
+      title: 'File for disability compensation',
+      form526RequiredIdentifiers: {
+        participantId: true,
+        birlsId: false,
+        ssn: true,
+        birthDate: true,
+        edipi: true,
+      },
+    };
+
+    it('returns the missing birlsId OK message', () => {
+      const tree = render(<Missing526Identifiers {...props} />);
+      const messageParagraph = tree.container.querySelector('p');
+
+      expect(messageParagraph.textContent).match(
+        /It's OK if you don't know your BIRLS ID./,
+      );
+    });
+  });
+
+  describe('edipi Missing', () => {
+    const props = {
+      title: 'File for disability compensation',
+      form526RequiredIdentifiers: {
+        participantId: true,
+        birlsId: true,
+        ssn: true,
+        birthDate: true,
+        edipi: false,
+      },
+    };
+
+    it('returns the missing edipi OK message', () => {
+      const tree = render(<Missing526Identifiers {...props} />);
+      const messageParagraph = tree.container.querySelector('p');
+
+      expect(messageParagraph.textContent).match(
+        /It's OK if you don't know your EDIPI./,
+      );
+    });
+  });
+
+  describe('edipi and birlsId Missing', () => {
+    const props = {
+      title: 'File for disability compensation',
+      form526RequiredIdentifiers: {
+        participantId: true,
+        birlsId: false,
+        ssn: true,
+        birthDate: true,
+        edipi: false,
+      },
+    };
+
+    it('returns the missing edipi and birls OK message', () => {
```

Comment:
birlsId

Reply:
Will fix and link in upcoming PR

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/32569